blueprints/auto-scaling/pod-auto-scaler/config.libsonnet

```diff
@@ -1,9 +1,34 @@
 local autoScalingDefaults = import '../common/config-defaults.libsonnet';
 
+/**
+* @schema (scaling_driver.scale_out.enabled: bool) Enables the driver to do scale out of the resource.
```

Comment:
Seems to be mismatch between schema and defaults below: `scaling_driver` vs `scaling_threshold` at https://github.com/fluxninja/aperture/pull/2224/files?diff=split&w=0#diff-531ae09bddf21b0099d51bed37dcb3b59003edc91f1977d5c949e1e3976faa5bR33

Reply:
Oh yes, I renamed the field later but missed to update here. Will update it